sequencer/eth/rollup.go

```diff
@@ -560,6 +561,39 @@ func (c *RollupClient) RollupEventsByBlock(blockNum int64,
 	return &rollupEvents, nil
 }
 
+// RollupForgeBatch is the interface to call the smart contract function
+func (c *RollupClient) RollupForgeBatch(args *RollupForgeBatchArgs, auth *bind.TransactOpts) (tx *types.Transaction, err error) {
+	if auth == nil {
+		auth, err = c.client.NewAuth()
+		if err != nil {
+			return nil, err
+		}
+		auth.GasLimit = 1000000
+	}
+
+	// nLevels := c.consts.Verifiers[args.VerifierIdx].NLevels //check verifiers
+
+	newLastIdx := big.NewInt(int64(args.NewLastIdx))
+
+	// var l1TxData []byte
+	// for i := 0; i < len(args.L1UserTxs); i++ {
+	// 	l1User := args.L1UserTxs[i]
+	// 	bytesl1User, err := l1User.BytesDataAvailability(uint32(nLevels))
+	// 	if err != nil {
+	// 		return nil, err
+	// 	}
+	// 	l1TxData = append(l1TxData, bytesl1User[:]...)
+	// }
```

Comment:
Could you elaborate the reason why this part should be commented out?

Reply:
We are not sending l1TxData to smart contract as compared to hermez smart contract which expect L1txData. so basically we dont have use for this yet, but only when tested with smart contract I can fully confirm that without l1txData we will be perform forge operation.